AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/IncomingAsyncSubscriptionEventToAnyModelMapper.swift

```diff
@@ -45,27 +45,15 @@ final class IncomingAsyncSubscriptionEventToAnyModelMapper: Subscriber, Cancella
         subscription.request(.max(1))
     }
 
-    func receive(_ input: IncomingAsyncSubscriptionEventPublisher.Event) -> Subscribers.Demand {
-        log.verbose("\(#function): \(input)")
-
-        switch input {
-        case .completed:
-            log.debug("Received completed event: \(input)")
-            modelsFromSubscription.send(completion: .finished)
-        case .failed(let apiError):
-            let dataStoreError = DataStoreError.api(apiError)
-            log.error(error: dataStoreError)
-            modelsFromSubscription.send(completion: .failure(dataStoreError))
-        case .inProcess(let subscriptionEvent):
-            dispose(of: subscriptionEvent)
-        default:
-            break
-        }
+    func receive(_ subscriptionEvent: IncomingAsyncSubscriptionEventPublisher.Event) -> Subscribers.Demand {
+        log.verbose("\(#function): \(subscriptionEvent)")
+        dispose(of: subscriptionEvent)
         return .max(1)
     }
 
     func receive(completion: Subscribers.Completion<DataStoreError>) {
         log.info("Received completion: \(completion)")
+        modelsFromSubscription.send(completion: completion)
```

Comment:
There is a comment on line 65/66 referring to AsyncEvent Completion/Error.  I don't think this is relevant anymore given that we are just receiving `SubscriptionEvent`

Reply:
Thanks for catching, will change